plugin-server/src/utils/db/db.ts

```diff
@@ -225,6 +234,55 @@ export class DB {
         })
     }
 
+    public async cachedPostgresQuery<R extends QueryResultRow = any, I extends any[] = any[]>(
+        queryString: string,
+        values: I | undefined,
+        tag: string,
+        client?: PoolClient,
```

Comment:
can we make this optional one be key-value, so folks don't need to say 'undefined' when changing ttl